crates/ra_ide_api/src/call_info.rs

```diff
@@ -93,8 +93,8 @@ enum FnCallNode {
 }
 
 impl FnCallNode {
-    fn with_node(syntax: &SyntaxNode, offset: TextUnit) -> Option<FnCallNode> {
-        ancestors_at_offset(syntax, offset).find_map(|node| {
+    fn with_node(syntax: &SyntaxNode) -> Option<FnCallNode> {
+        syntax.ancestors().find_map(|node| {
```

Comment:
@matklad Is this change equivalent?

Reply:
They are completely different: the new one will return an ancessor of `syntax`, the old one might return it's child. I *think* the old one more precise in cases like `foo(<|>bar())`